### PR TITLE
fix the short descriptions of all helm commands

### PR DESCRIPTION
**Pull request by tariq1890**
This PR Capitalizes the first word of every sentence in all short descriptions. Suggesting this for two reasons

i) Some short descriptions of the helm subcommands have this and some don't. This PR brings consistency.
ii) Kubectl follows this convention as well.

Screen Shot 2019-05-01 at 11 11 29 AM

### CODE REVIEW

**Review by hickeyma — CHANGES_REQUESTED** [review]
@tariq1890 Thanks for tidying this up.

Some feedback as follows:
- "rollback    Roll back a release to a previous revision" —>  "rollback    Rollback a release to a previous revision"
- Flags also need to be capitalized. Here are some examples:
```console
Flags:
      --debug                           enable verbose output
  -h, --help                            help for helm
      --home string                     location of your Helm config. Overrides $HELM_HOME (default "/home/usr1/.helm")
      --host string                     address of Tiller. Overrides $HELM_HOST
      --kube-context string             name of the kubeconfig context to use
      --kubeconfig string               absolute path to the kubeconfig file to use
      --tiller-connection-timeout int   the duration (in seconds) Helm will wait to establish a connection to tiller (default 300)
      --tiller-namespace string         namespace of Tiller (default "kube-system")

Flags:
      --description string    specify a description for the release
      --dry-run               simulate a delete
  -h, --help                  help for delete
      --no-hooks              prevent hooks from running during deletion
      --purge                 remove the release from the store and make its name free for later use
      --timeout int           time in seconds to wait for any individual Kubernetes operation (like Jobs for hooks) (default 300)
      --tls                   enable TLS for request
      --tls-ca-cert string    path to TLS CA certificate file (default "$HELM_HOME/ca.pem")
      --tls-cert string       path to TLS certificate file (default "$HELM_HOME/cert.pem")
      --tls-hostname string   the server name used to verify the hostname on the returned certificates from the server
      --tls-key string        path to TLS key file (default "$HELM_HOME/key.pem")
      --tls-verify            enable TLS for request and verify remote
```
- Other parts need to be capitalized also:
```console
Common actions from this point include:

- helm search:    search for charts
- helm fetch:     download a chart to your local directory to view
- helm install:   upload the chart to Kubernetes
- helm list:      list releases of charts

Environment:

- $HELM_HOME:           set an alternative location for Helm files. By default, these are stored in ~/.helm
- $HELM_HOST:           set an alternative Tiller host. The format is host:port
- $HELM_NO_PLUGINS:     disable plugins. Set HELM_NO_PLUGINS=1 to disable plugins.
- $TILLER_NAMESPACE:    set an alternative Tiller namespace (default "kube-system")
- $KUBECONFIG:          set an alternative Kubernetes configuration file (default "~/.kube/config")
- $HELM_TLS_CA_CERT:    path to TLS CA certificate used to verify the Helm client and Tiller server certificates (default "$HELM_HOME/ca.pem")
- $HELM_TLS_CERT:       path to TLS client certificate file for authenticating to Tiller (default "$HELM_HOME/cert.pem")
- $HELM_TLS_KEY:        path to TLS client key file for authenticating to Tiller (default "$HELM_HOME/key.pem")
- $HELM_TLS_ENABLE:     enable TLS connection between Helm and Tiller (default "false")
- $HELM_TLS_VERIFY:     enable TLS connection between Helm and Tiller and verify Tiller server certificate (default "false")
- $HELM_TLS_HOSTNAME:   the hostname or IP address used to verify the Tiller server certificate (default "127.0.0.1")
- $HELM_KEY_PASSPHRASE: set HELM_KEY_PASSPHRASE to the passphrase of your PGP private key. If set, you will not be prompted for the passphrase while signing helm charts
```

**Comment by tariq1890** [comment]
@hickeyma Done

**Comment by hickeyma** [comment]
@tariq1890 Can you rebase and I'll review again then?

**Comment by tariq1890** [comment]
@hickeyma @bacongobbler PTAL. Thanks

File: cmd/helm/search.go [hunk 1]
```diff
@@ -64,10 +64,10 @@ func newSearchCmd(out io.Writer) *cobra.Command {
 	}
 
 	f := cmd.Flags()
-	f.BoolVarP(&sc.regexp, "regexp", "r", false, "use regular expressions for searching")
-	f.BoolVarP(&sc.versions, "versions", "l", false, "show the long listing, with each version of each chart on its own line")
-	f.StringVarP(&sc.version, "version", "v", "", "search using semantic versioning constraints")
-	f.UintVar(&sc.colWidth, "col-width", 60, "specifies the max column width of output")
+	f.BoolVarP(&sc.regexp, "regexp", "r", false, "Sse regular expressions for searching")
```
Comment (hickeyma) [inline cmd/helm/search.go]: :s/Sse/Use

**Review by hickeyma — CHANGES_REQUESTED** [review]
Looking good except for small nit. Could you also squash the changes to be able to test it easier?

File: cmd/helm/search.go (hunk 1 above)
Comment (tariq1890) [reply cmd/helm/search.go]: Whoops! Thanks for catching that :)

Comment (tariq1890) [reply cmd/helm/search.go]: I have squashed the changes as well.

**Comment by tariq1890** [comment]
Looks like CircleCI is acting up.

**Review by thomastaylor312 — APPROVED** [review]
LGTM. @hickeyma feel free to merge when you are done reviewing

**Review by hickeyma — CHANGES_REQUESTED** [review]
@tariq1890 Thanks for your work on this,. It is nearly there. Here are some suggested updates:

- In Flag, `-h, --help   help for <command>` --> `-h, --help   Help for <command>`
- In Global Flag, `--tiller-connection-timeout int   The duration (in seconds) Helm will wait to establish a connection to tiller (default 300)` --> `--tiller-connection-timeout int   The duration (in seconds) Helm will wait to establish a connection to Tiller (default 300)`
- In `helm dependency`,  `list        list the dependencies for the given chart` --> `list        List the dependencies for the given chart`

**Comment by tariq1890** [comment]
I've addressed your review comments except for this
> In Flag, -h, --help help for <command> --> -h, --help Help for <command>
I am trying to figure out an elegant way to do this  as the `help` flag is auto-created by `cobra`.

**Comment by tariq1890** [comment]
@hickeyma Any thoughts on this? One way to get around this is by manually adding the `help` flag to every helm subcommand.

**Review by hickeyma — APPROVED** [review]
> One way to get around this is by manually adding the help flag to every helm subcommand.

I think we can leave this.

Thanks @tariq1890 for your efforts in aligning this with other CLIs in the k8s ecosystem.

**Pull request merged**